tools/crm_mon.c

```diff
@@ -515,6 +553,12 @@ main(int argc, char **argv)
         one_shot = TRUE;
     }
 
+    /* to enable stonith-connection when called via some application like pcs
+     * set environment-variable FENCE_HISTORY to desired level
+     * so you don't have to modify this application
+     */
+    /* fence_history_level = crm_atoi(getenv("FENCE_HISTORY"), "0"); */
+
```

Comment:
Since the relevant code was removed, we should remove this comment block too

Reply:
That is now the relevant code. Uncomment this one line and you have it as before.

Reply:
Whoops, I misread :)